src/app/games/epaminondas/epaminondas.component.ts

```diff
@@ -25,10 +25,10 @@ import { MGPOptional } from 'src/app/utils/MGPOptional';
 export class EpaminondasComponent extends RectangularGameComponent<EpaminondasRules,
                                                                    EpaminondasMove,
                                                                    EpaminondasState,
-                                                                   Player,
+                                                                   PlayerOrNone,
                                                                    EpaminondasLegalityInformation>
 {
-    public NONE: Player = Player.NONE;
+    public NONE: PlayerOrNone = PlayerOrNone.NONE;
```

Comment:
elle est utilisé celle là in fine ?

Reply:
yep, dans le HTML: `            <circle *ngIf="board[y][x] !== NONE"`